packages/components/src/core/DropdownMenu/index.tsx

```diff
@@ -86,14 +99,36 @@ const DropdownMenu = <
     options,
     onClickAway = noop,
     ClickAwayListenerProps,
-    width = 160,
+    width = MINIMUM_DROPDOWN_MENU_POPPER_WIDTH,
     ...rest
   } = props;
 
   const isMultiColumn = options && !!options[0] && "options" in options[0];
-  const singleColumnPadding = `${theme.app?.spacing?.xs}px 0 ${theme.app?.spacing?.xs}px ${theme.app?.spacing?.xs}px !important`;
-  const multiColumnNoSearchOrTitlePadding = `${theme.app?.spacing?.l}px ${theme.app?.spacing?.xxs}px ${theme.app?.spacing?.l}px ${theme.app?.spacing?.l}px !important`;
-  const multiColumnWithSearchOrTitlePadding = `${theme.app?.spacing?.xs}px 0 ${theme.app?.spacing?.l}px ${theme.app?.spacing?.xs}px !important`;
+
+  // (masoudmanson): The DropdownMenu's Popper component should have a minimum
+  // width of MINIMUM_DROPDOWN_MENU_POPPER_WIDTH pixels if the DropdownMenu is
+  // a single column dropdown.However, for larger multi-column autocompletes,
+  // the width should be set to 'auto' to accommodate the expanded layout.
+  const dropdownMenuPopperSx = useMemo(() => {
+    return {
+      ...PopperBaseProps?.sx,
+      width:
+        isMultiColumn || width < MINIMUM_DROPDOWN_MENU_POPPER_WIDTH
+          ? "auto"
+          : width,
+    };
+  }, [PopperBaseProps?.sx, isMultiColumn, width]);
+
+  const DefaultPopperBaseProps = {
```

Comment:
Ah sorry this has to be lifted outside of the component function level to prevent a recreation of this object every time the component function is called. And since it's constant, we can use ALL_CAPS too!

E.g.,

```ts
const DEFAULT_POPPER_BASE_PROPS = {
  disabledPortal: true,
}

const DropdownMenu = () => {
 ...
 return (
   <...>
     <<Autocomplete
                 ...
                {...rest}
                PopperBaseProps={DEFAULT_POPPER_BASE_PROPS}
                ...
              />
   </...>
 )
}
```

Reply:
Ah right, it also had a typo `disabledPortal` -> `disablePortal` 😱

Reply:
Ooh! I didn't catch that ahaha great eye!